sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java

```diff
@@ -196,7 +233,9 @@ private BeamRelNode convertToBeamRelInternal(String sql, QueryParameters queryPa
     RelMetadataQuery.THREAD_PROVIDERS.set(
         JaninoRelMetadataProvider.of(root.rel.getCluster().getMetadataProvider()));
     root.rel.getCluster().invalidateMetadataQuery();
-    return (BeamRelNode) plannerImpl.transform(0, desiredTraits, root.rel);
+    BeamRelNode beamRelNode = (BeamRelNode) plannerImpl.transform(0, desiredTraits, root.rel);
+    LOG.info("BEAMPlan>\n" + RelOptUtil.toString(beamRelNode));
```

Comment:
Nit: do we need this LOG.info?

Reply:
Not really, but it may be useful someday. I actually left this in from your original PR (#12398).

Reply:
I guess that was added for the purpose of debugging. 

Keep this LOG.info is fine. I find that in CalciteQueryPlanner there is also a such LOG.info.

Reply:
I guess LOG.debug would probably be better. I might change both planners in a separate PR.